src/components/ProfileCard.jsx

```diff
@@ -0,0 +1,12 @@
+// eslint-disable-next-line react/prop-types
```

Comment:
Lets put the actual prop-types declarations, i know it makes it look a little ehh but it'll make it easier next time for the people that are going to be working on the page to understand what kind of data the parameters are going to be in this function